_include/q/gradle-quickstart.sh

```diff
@@ -38,14 +38,22 @@ function mkPackage() {
   echo ${x//./\/}
 }
 
+function extract_parameter() {
```

Comment:
What about creating a _utils.sh script within `_include/q/` and move this function to this script. We're using it in multiple locations. That would reduce the code redundancy. Also some other functions (from `gradle-quickstart.sh` and `sbt-quickstart.sh` can be collected in that file). WDYT?